compiler/src/iree/compiler/Codegen/Common/test/verify_tuning_specs.mlir

```diff
@@ -51,3 +51,9 @@ module @foo_module attributes { transform.with_named_sequence } {
   transform.named_sequence @foo(%arg0: !transform.any_op {transform.readonly})
     attributes { iree_codegen.tuning_spec_entrypoint } {}
 }
+
+// -----
+
+// expected-error @+1{{The default tuning specification must include an operation with the symbol name '__kernel_config'}}
+module @iree_default_tuning_spec attributes { iree_codegen.default_tuning_spec } {
+}
```

Comment:
We should also have a test for when there's some other op with named `__kernel_config`, e.g., `func.func`.